runtime.c

```diff
@@ -209,15 +209,22 @@ print_help()
 {
 // TODO:   "--appimage-list                 List content from embedded filesystem image\n"
     printf("AppImage options:\n\n"
-           "--appimage-extract              Extract content from embedded filesystem image\n"
-           "--appimage-help                 Print this help\n"
-           "--appimage-mount                Mount embedded filesystem image and\n"
-           "                                print mount point and wait for kill with Ctrl-C\n"
-           "--appimage-offset               Print byte offset to start of\n"
-           "                                embedded filesystem image\n"
-           "--appimage-signature            Print digital signature embedded in AppImage\n"
-           "--appimage-updateinfo[rmation]  Print update info embedded in AppImage\n"
-           "--appimage-version              Print version of AppImageKit\n"
+           "  --appimage-extract              Extract content from embedded filesystem image\n"
```

Comment:
Why did the indentation of this and the following lines change?

Reply:
I changed it on purpose because I thought indenting the options looks nicer. Can revert it if you don't like it, though

Reply:
Personally I liked it nicely aligned better, looked cleaner to me. Is there a (quasi) standard? See how the `appimagetool` help, for example is formatted (there we are using glib to do it, which is not an option here.)

Reply:
Sure, I will revert that. I just added the indentation this time because I added the other "Portable options" section and I thought it looked better if the options were indented to their corresponding section

Reply:
![virtualbox_ubuntu 14 04_21_11_2017_09_03_20](https://user-images.githubusercontent.com/3248441/33061050-da2ca074-ce9a-11e7-8747-b8aedcd21792.png)
Actually, appimagetool uses 2 whitespaces :)

Reply:
OK, sorry for the confusion caused and thanks for checking with `appimagetool`. Consistency is key. Note there are indentions in the left but _not_ in the right column.

Reply:
![image](https://user-images.githubusercontent.com/3248441/33089786-03a2a104-cef2-11e7-9d6d-18cfc05ad0e4.png)
Do you mean I should indent the last two lines to be in the same column as the right column of the options section?